README.md

```diff
@@ -31,10 +31,13 @@ where Rails.config.zipkin_tracer or config is a hash that can contain the follow
  * `:whitelist_plugin` - plugin function which recieves Rack env and will force sampling if it returns true
  * `:zookeeper` - plugin function which uses zookeeper and kafka instead of scribe as the transport
  * `:logger` - A logger class following the standard's library Logger interface (Log4r, Rails.logger, etc).
+ * `json_api_host` - hostname with protocol of a zipkin api instance (e.g. `https://zipkin.example.com`) to use JSON over HTTP instead of Scribe or Kafka
+ * `traces_buffer` (default: 10) - the number of annotations stored until automatic flush
```

Comment:
Could this number be a bit too small? partial spans are kindof a pain. Ex. in brave, it is 10 spans as opposed to 10 annotations. 

In finagle, it is much nicer, as it goes based on the size of the data 1MB. Since this is json text, we might be able to flush based on size sooner or later.